source/documents/view.txt

```diff
@@ -42,7 +42,7 @@ to use:
    .. tab:: List View
```

Comment:
(suggestion) 
It took me a minute to understand that the contents of the page changed for _the rest of the page_ depending on what tab you select here. It might be worth mentioning. 
Happy to chat if you want!

Reply:
I changed the wording to make it more clear that the selected view is shown in all examples